docs/content/preview/drivers-orms/nodejs/_index.md

```diff
@@ -13,17 +12,24 @@ menu:
 isTocNested: true
 showAsideToc: true
 ---
+
 The following projects are recommended for implementing Node applications using the YugabyteDB YSQL API.
 
-| Project | Type | Support | Examples |
-| :------ | :--- | :------ | :------- |
-| [node-postgres](postgres-node-driver) | Node.JS Driver | Full | [Hello World](/preview/quick-start/build-apps/nodejs/ysql-pg/) <br />[CRUD](postgres-node-driver) |
-| [Sequelize](sequelize) | ORM | Full | [Hello World](/preview/quick-start/build-apps/nodejs/ysql-sequelize/) <br />[CRUD](sequelize) |
-| [Prisma](prisma) | ORM | Full | [Hello World](/preview/quick-start/build-apps/nodejs/ysql-prisma/) <br /> [CRUD](prisma) |
+## Supported Projects
+
+| Project | Documentation and Guides | Latest Driver Version | Supported YugabyteDB Version |
+| :------ | :----------------------- | :----------------------- | :--------------------|
+| node-postgres Driver| [Documentation](postgres-node-driver) <br /> [Hello World App](../../quick-start/build-apps/nodejs/ysql-pg/) | [8.7.3](https://www.npmjs.com/package/pg) | 2.6 and above |
+
+| Project | Documentation and Guides | Example Apps |
+| :------ | :----------------------- | :----------- |
+| Sequelize | [Documentation](sequelize) <br /> [Hello World App](../../quick-start/build-apps/nodejs/ysql-sequelize/) | [Sequelize ORM App](https://github.com/yugabyte/orm-examples/tree/master/node/sequelize) |
```

Comment:
The `Sequelize` row needs to be removed and all the links and examples which are there in this Sequelize row are for `Sequelize Yugabytedb`  which uses YugabyteDB specific package for sequelize as for now. `Sequelize` page in `Use an ORM` section explains the use with core sequelize with some limitations with YugabyteDB, so that is not required to be mentioned in this table.
CC: @ashetkar